source/includes/steps-create-local-master-key.yaml

```diff
@@ -0,0 +1,68 @@
+title: Create the local master key
+ref: create-local-master-key
+level: 4
+content: |
+  Generate the local master key and save it to a file. You should only need to run this once:
```

Comment:
Let's be a little more imperative. Some users need the extra hand holding.

```rst
The following script generates a local master key and saves it to a file called ``master-key.txt``
in the same directory as the script. Copy the code and run it to generate the master key for this
application.
```

Also, do we need to specify that it should only run once? I would say we don't unless there is a significant downside to running it twice (e.g. crazy errors) or we see that users commonly do run it multiple times when they don't need to.

Reply:
Can we start this step with a brief overview of the anatomy of a master key? It looks like it has 96 bytes but a little more info would be nice (e.g. "it's 96 randomly generated bytes that have no relation to eachother")

```rst
Envelope encryption uses 96 byte master keys where each byte can be any random value. To perform
CSFLE, you must generate a master key.
```

Reply:
Sounds good, will add!